edgemesh/pkg/proxy/proxy.go

```diff
@@ -150,13 +150,13 @@ func (p *Proxier) readAndCleanRule() {
 	for scan.Scan() {
 		serverString := scan.Text()
 		if strings.Contains(serverString, "-o") {
-			p.iptables.DeleteRule(utiliptables.TableNAT, utiliptables.ChainOutput, strings.Split(serverString, " ")...)
+			_ = p.iptables.DeleteRule(utiliptables.TableNAT, utiliptables.ChainOutput, strings.Split(serverString, " ")...)
```

Comment:
I don't understand those change and other occurrences as well.

Reply:
This pr is to fix errcheck lint issue, i log err if needed i think. So for some part, just use `_` to fix lint issue.